src/posix/net.cpp

```diff
@@ -162,13 +162,20 @@ CurlHttpIO::CurlHttpIO()
     curl_multi_setopt(curlm[GET], CURLMOPT_SOCKETDATA, this);
     curl_multi_setopt(curlm[GET], CURLMOPT_TIMERFUNCTION, download_timer_callback);
     curl_multi_setopt(curlm[GET], CURLMOPT_TIMERDATA, this);
+#ifdef _WIN32
+    curl_multi_setopt(curlm[GET], CURLMOPT_MAXCONNECTS, 200);
```

Comment:
cURL just discards the oldest cached connection.
I have increased the limit on Windows because I noticed that it was already happening after few transfers. I haven't increased the limit for other platforms because limits of the operating system could conflict (specially OS X).